### DISCUSSION TITLE
Link to specific form entry?

### DISCUSSION THREAD

**Discussion by Gehmalbierholen123e** [Budibase/budibase#10335] [Help]
Hello,
I have a form block with different fileds on one screen, where I can enter certain informations, f.e. "Friends".
On another Screen I have a table block to show off all entries. I have also activated the Details Side Panel where all Information about one person is shown.

Now I want to do the following things:
1. The "Friends" field is a "Long Form Field". I want to replace it with a drop down, where I can see all the people entered into the databse (first name and last name are two seperate Text fields in the form) and select the appropriate persons.

2. I want the details side panel display all the selected friends (their names) one below the other (This is only optional) and have them link to the corresponding entry in the database (where the "Link" opens the details side panel for that person).

Is that possible and if so, how?
If there is any more questions, ask.

Thanks in advance!

**Comment by melohagan** [comment]
Hey @Gehmalbierholen123e 

I'm not entirely visualising this one - could you share some screenshots of your screens please?

**Reply by Gehmalbierholen123e** [reply]
> I'm not entirely visualising this one - could you share some screenshots of your screens please?

Hello,
I have made some Screenshots with annotations to make it more clear. Hope that helps :)

![2023-04-18 22_23_35-Ferienjob_Supermarkt_Bewerbungsschreiben - Word](https://user-images.githubusercontent.com/128325093/232897434-0fb91e99-abd9-4ef0-8197-8e80079979fd.png)

![2023-04-18 22_11_56-Ferienjob_Supermarkt_Bewerbungsschreiben - Word](https://user-images.githubusercontent.com/128325093/232897614-7f36a844-7f94-480a-b2a8-a9d2055f3b2e.png)

![2023-04-18 22_17_07-](https://user-images.githubusercontent.com/128325093/232897648-8e2d3838-16e1-4a9f-aad8-35a40a82377a.png)

![2023-04-18 22_20_03-gddatabase](https://user-images.githubusercontent.com/128325093/232897759-d6039eb7-8df3-4a1a-b03e-228b6775c9c9.png)

![2023-04-18 22_24_41-Budibase](https://user-images.githubusercontent.com/128325093/232897828-8804dd19-c34e-419f-ab5d-a83836daf34e.png)

Thank you for your time to help!

**Comment by melohagan** [comment]
Hey @Gehmalbierholen123e 

Thanks for the screenshots. I shall address the Freunde list first. 

I would add a Repeater to pull the friends from the relationship and nest a button for each friend that opens the side panel. You may also need to setup your own app state for handling the friends side panel.
If you don't want to use a button, then you could add a container with an *On Click* action that has a paragraph within.

For the second point, when displaying relationships you must use the **Relationship Picker**, and not the multi-select.

Here's some docs that may be useful:

 - https://docs.budibase.com/docs/app-state
 - https://docs.budibase.com/docs/side-panel
 - https://docs.budibase.com/docs/filter-by-relationships
 - https://docs.budibase.com/docs/fetching-data#displaying-relational-data
